pkg/reporter/summary.go

```diff
@@ -248,9 +248,11 @@ func (r *summaryReporter) renderRemediationAdvice() {
 		insight := utils.SafelyGetValue(sp.pkg.Insights)
 
 		tbl.AppendRow(table.Row{
+			sp.ecosystem,
```

Comment:
Looks good. You need to add an appropriate header row as well for this additional column. Have a look at https://github.com/safedep/vet/blob/main/pkg/reporter/summary.go#L242

Reply:
So you are saying I have to add a new column as well. Right?

Reply:
Yes. You need to add a new column for the header row as well.